src/sql/src/func.rs

```diff
@@ -1610,6 +1610,15 @@ lazy_static! {
                 // encoding id for UTF8 (6) is provided, otherwise return 'NULL'.
                 params!(Int64) => sql_impl_func("CASE WHEN $1 = 6 THEN 'UTF8' ELSE NULL END"), 1597;
             },
+            // pg_get_constraintdef gives more info about a constraint with in the `pg_constraint`
+            // view. It currently returns no information as the `pg_constraint` view is empty in
+            // materialize
+            "pg_get_constraintdef" => Scalar {
+                params!(Oid) => {
+                    UnaryFunc::NotImplemented("pg_get_constraintdef".to_string())}, 1598;
+                params!(Oid, Bool) => {
+                    BinaryFunc::NotImplemented("pg_get_constraintdef".to_string())}, 1599;
```

Comment:
how did you choose these OIDs? I'm not sure how we choose them.

Reply:
I'm not sure what you mean

Reply:
Oids are unique identifiers in postgres. Every thing has one and they are globally unique: tables, types, operators, functions, etc. Here's the correct oids:

```
postgres=# select oid, proname, proargtypes from pg_proc where proname like 'pg_get_constraint%';
 oid  |       proname        | proargtypes 
------+----------------------+-------------
 1387 | pg_get_constraintdef | 26
 2508 | pg_get_constraintdef | 26 16
```

So 1387 for the single arg, and 2508 for the two argument.